server.js

```diff
@@ -1,5 +1,8 @@
 const express = require('express');
 const app = express()
+const cors = require('cors');
+
+app.use(cors());
```

Comment:
Using CORS in the express API will allow us to make fetch requests from the front-end application. 👍🏼